filament/src/components/RenderableManager.h

```diff
@@ -114,6 +115,8 @@ class FRenderableManager : public RenderableManager {
     inline void setReceiveShadows(Instance instance, bool enable) noexcept;
     inline void setScreenSpaceContactShadows(Instance instance, bool enable) noexcept;
     inline void setCulling(Instance instance, bool enable) noexcept;
+    inline void setFogEnabled(Instance instance, bool enable) noexcept;
+    inline bool getFogEnabled(RenderableManager::Instance instance) const noexcept;
```

Comment:
Why qualify Instance as `RenderableManager::Instance` here and not in `setFogEnabled`?

Reply:
oh it's just a mistake. thanks.